multi-node/generic/controller-install.sh

```diff
@@ -393,6 +393,109 @@ EOF
 EOF
     }
 
+    local TEMPLATE=/srv/kubernetes/manifests/tpm-policy.json
+    [ -f $TEMPLATE ] || {
+        echo "TEMPLATE: $TEMPLATE"
+        mkdir -p $(dirname $TEMPLATE)
+        cat << EOF > $TEMPLATE
+{
+  "kind": "ThirdPartyResource",
+  "apiVersion": "extensions/v1beta1",
+  "metadata": {
+    "name": "policy.tpm.coreos.com"
+  },
+  "description": "Policy describing TPM attestation measurements",
+  "versions": [
+    {
+      "name": "v1"
+    }
+  ]
+}
+EOF
+    }
+
+    local TEMPLATE=/srv/kubernetes/manifests/tpm.json
+    [ -f $TEMPLATE ] || {
+        echo "TEMPLATE: $TEMPLATE"
+        mkdir -p $(dirname $TEMPLATE)
+        cat << EOF > $TEMPLATE
+{
+  "kind": "ThirdPartyResource",
+  "apiVersion": "extensions/v1beta1",
+  "metadata": {
+    "name": "tpm.coreos.com"
+  },
+  "description": "TPM data",
```

Comment:
Can we add more to this? Is it accurate to say "TPM [measurements,data] [read,compiled,generated,measured] during boot"